libs/asset-viewer/src/lib/components/asset-viewer-page/asset-viewer-page.component.scss

```diff
@@ -14,6 +14,15 @@
   grid-template-columns: auto 1fr auto;
 }
 
+.loading-bar {
+  position: absolute;
+  inset: 0;
+  z-index: 100;
+  display: grid;
```

Comment:
No reason to make this a grid, this can just be a `block`.

+ remove the duplicate `position: relative;` from the `:host` selector (line 4).